### PR TITLE
Add Add ForceAuditChecksCmd to reset auditor checks last run time

### PR DESCRIPTION
**Pull request by karanmehta93**
Descriptions of the changes in this PR:

### Motivation

Reset the last run time for auditor checks. Reboot auditor leader immediately to trigger the checks. This is mostly used for testing infra and get the sanity check of a cluster on demand.

### Changes

Added a new shell command and tests

### CODE REVIEW

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java [hunk 1]
```diff
@@ -1799,6 +1804,76 @@ public int runCmd(CommandLine cmdLine) throws Exception {
...
+
+            if (checkAllLedgersCheck || placementPolicyCheck  || replicasCheck) {
+                runFunctionWithLedgerManagerFactory(bkConf, mFactory -> {
+                    try {
+                        try (LedgerUnderreplicationManager underreplicationManager =
+                                     mFactory.newLedgerUnderreplicationManager()) {
+                            // Arbitrary value of 21 days chosen since current freq of all checks is less than 21 days
+                            long time = System.currentTimeMillis() - (21 * 24 * 60 * 60 * 1000);
+                            if (checkAllLedgersCheck) {
+                                LOG.info("Resetting CheckAllLedgersCTime to : " + time);
```
Comment (eolivelli) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java:1851]: can we ALSO print `time` formatted for humans ?

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java [hunk 2]
```diff
@@ -1799,6 +1804,76 @@ public int runCmd(CommandLine cmdLine) throws Exception {
...
+                                underreplicationManager.setCheckAllLedgersCTime(time);
+                            }
+                            if (placementPolicyCheck) {
+                                LOG.info("Resetting PlacementPolicyCheckCTime to : " + time);
```
Comment (eolivelli) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java:1855]: can we ALSO print `time` formatted for humans ?

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java [hunk 3]
```diff
@@ -1799,6 +1804,76 @@ public int runCmd(CommandLine cmdLine) throws Exception {
...
+                                underreplicationManager.setPlacementPolicyCheckCTime(time);
+                            }
+                            if (replicasCheck) {
+                                LOG.info("Resetting ReplicasCheckCTime to : " + time);
```
Comment (eolivelli) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java:1859]: can we ALSO print `time` formatted for humans ?

**Review by eolivelli — CHANGES_REQUESTED** [review]
very good !
thanks for this useful tool.
I left a comment.
Also, I believe that this patch does not add the same command to "bkctl" but only to BookieShell.
BookieShell is the old tool, all of the new commands should also be added to bkctl.
It is not a matter of duplicating code, but only to register the handler also in bkctl runtime, take any other command as example

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java (hunk 3 above)
Comment (karanmehta93) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java]: Updated it.

**Review by sijie — APPROVED** [review]

**Review by eolivelli — APPROVED** [review]
LGTM

**Pull request merged**